Test Some Time Failed

This issue highlights a problem in which testing sometimes fails, as could be seen in this log:

```
Internal ctest changing into directory: /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/build
Test project /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/build
    Start 1: Testing sources formatting
1/3 Test #1: Testing sources formatting ...............   Passed    1.39 sec
    Start 2: Testing include directories formatting
2/3 Test #2: Testing include directories formatting ...***Failed    1.38 sec
-- Getting the original source file hashes
-- Copying the ugly source files
-- Configuring sample project
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (0.6s)
-- Generating done (0.0s)
-- Build files have been written to: /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/build
-- Building sample project
[ 25%] Built target sample_format
[ 50%] Building CXX object CMakeFiles/sample.dir/src/fibonacci.cpp.o
[ 75%] Building CXX object CMakeFiles/sample.dir/src/is_odd.cpp.o
[100%] Linking CXX static library libsample.a
[100%] Built target sample
-- Comparing the source file hashes
CMake Error at /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:57 (message):
  File hash of include/sample/fibonacci.hpp is different: got
  795823310785776c79d67f9c18a74d18, should be
  5719e7434c3ed8e41b109b7753c70f68
Call Stack (most recent call first):
  /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:73 (check_source_codes_format)



    Start 3: Testing file set headers formatting
3/3 Test #3: Testing file set headers formatting ......***Failed    1.33 sec
-- Getting the original source file hashes
-- Copying the ugly source files
-- Configuring sample project
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (0.5s)
-- Generating done (0.0s)
-- Build files have been written to: /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/build
-- Building sample project
[ 25%] Generating sample_format.lock
[ 25%] Built target sample_format
[ 50%] Building CXX object CMakeFiles/sample.dir/src/fibonacci.cpp.o
[ 75%] Building CXX object CMakeFiles/sample.dir/src/is_odd.cpp.o
[100%] Linking CXX static library libsample.a
[100%] Built target sample
-- Comparing the source file hashes
CMake Error at /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:57 (message):
  File hash of include/sample/fibonacci.hpp is different: got
  5719e7434c3ed8e41b109b7753c70f68, should be
  795823310785776c79d67f9c18a74d18
Call Stack (most recent call first):
  /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:83 (check_source_codes_format)




33% tests passed, 2 tests failed out of 3

Total Test time (real) =   4.11 sec

The following tests FAILED:
          2 - Testing include directories formatting (Failed)
          3 - Testing file set headers formatting (Failed)
Errors while running CTest
```

This issue seems to be caused by the testing running too fast, resulting in the modification of the source files for testing happening quickly. Consequently, the timestamp is not updating, and it does not trigger the format fix because it is considered that nothing has changed.

--- a/test/FixFormatTest.cmake
+++ b/test/FixFormatTest.cmake
@@ -15,7 +15,7 @@ function(check_source_codes_format)
 
   execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1)
 
-  message(STATUS "Copying the ugly source files")
+  message(STATUS "Copying the dirty source files")
   foreach(SRC ${ARG_SRCS})
     file(
       COPY_FILE

--- a/test/FixFormatTest.cmake
+++ b/test/FixFormatTest.cmake
@@ -13,6 +13,8 @@ function(check_source_codes_format)
     file(MD5 ${CMAKE_CURRENT_LIST_DIR}/sample/${SRC} ${SRC}_HASH)
   endforeach()
 
+  execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1)
+
   message(STATUS "Copying the ugly source files")
   foreach(SRC ${ARG_SRCS})
     file(

